packages-exp/auth-exp/src/platform_browser/strategies/redirect.ts

```diff
@@ -34,91 +34,163 @@ import { User, UserCredential } from '../../model/user';
 import { _withDefaultResolver } from '../popup_redirect';
 import { AbstractPopupRedirectOperation } from './abstract_popup_redirect_operation';
 
+/**
+ * Authenticates a Firebase client using a full-page redirect flow.
+ *
+ * @remarks
+ * To handle the results and errors for this operation, refer to {@link getRedirectResult}.
```

Comment:
Maybe add a comment that calling this will cause the page to leave your app entirely

Reply:
i think it's already fairly clear in the summary, can rinse later with eric